settings.gradle

```diff
@@ -26,7 +26,9 @@ buildCache {
 
     remote(gradleEnterprise.buildCache) {
         enabled = true
-        push = isCI
+        // Check access key presence to avoid build cache errors on PR builds when access key is not present
+        def accessKey = System.getenv("GRADLE_ENTERPRISE_ACCESS_KEY")
+        push = isCI && accessKey != null && !accessKey.isEmpty()
```

Comment:
If we do this for this repo, we should do it for the other repos, too. Of, if we only need/want one example, we should do it in the config samples repo (for groovy and Kotlin).

Reply:
Agreed, we should do this for all the repos.